server/aws-lsp-codewhisperer/src/language-server/chat/chatController.test.ts

```diff
@@ -121,11 +122,31 @@ describe('ChatController', () => {
             }),
         }
 
+        const mockWorkspace: Workspace = {
+            getWorkspaceFolder: sinon.stub(),
+            getTextDocument: sinon.stub(),
+            getAllTextDocuments: sinon.stub(),
+            fs: {
+                copyFile: sinon.stub(),
+                exists: sinon.stub(),
+                getFileSize: sinon.stub(),
+                getServerDataDirPath: sinon.stub(),
+                getTempDirPath: sinon.stub(),
+                readFile: sinon.stub(),
+                readdir: sinon.stub(),
+                isFile: sinon.stub(),
+                rm: sinon.stub(),
+                writeFile: sinon.stub(),
+                appendFile: sinon.stub(),
+                mkdir: sinon.stub(),
```

Comment:
Do we have to explicitly specify all the methods of `workspace`? Extending `workspace` will break the build in replica sync again. Maybe we can do something like [here](https://github.com/aws/language-servers/pull/635/files#diff-bb635788256abd858778bc03153a2d8b258179aa8c5a208a85a956b7b83bf9d5R26)